software/docker/kafka-can-log/configure

```diff
@@ -0,0 +1,6007 @@
+#! /bin/sh
```

Comment:
I don't think this is supposed to be committed. We should add/update `.gitignore`

Some reference: https://wiki.debian.org/Autoreconf

If you don't want to remember the autoreconf command, then it is somewhat popular to have a script called `autogen.sh` which is basically:

```bash
autoreconf --install || exit 1
```

Sometimes it calls `./configure` for you too. But, when `configure` has options, that isn't always the nicest.